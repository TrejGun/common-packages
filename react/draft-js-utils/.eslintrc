{
  "extends": [
    "@gemunion/eslint-config/presets/tsx",
    "@gemunion/eslint-config/tests/jest"
  ],
  "parserOptions": {
    "project": "./tsconfig.test.json"
  },
  "root": true
}
