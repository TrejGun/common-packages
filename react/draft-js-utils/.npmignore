/@types
/src
/tsconfig.json
/jest.config.json
/setup-jest.ts
**/*.test.*
