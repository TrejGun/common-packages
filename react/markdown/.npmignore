/src
/@types
/tsconfig.json
/tsconfig.test.json
/*.stories.*
