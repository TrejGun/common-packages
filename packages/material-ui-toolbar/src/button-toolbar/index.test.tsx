import React from "react";
import {render, cleanup} from "@testing-library/react";
import {MuiThemeProvider, createTheme} from "@material-ui/core";

import {ButtonToolbar} from "./";

afterEach(cleanup);

describe("<ButtonToolbar />", () => {
  it("renders component", () => {
    const {asFragment} = render(
      <MuiThemeProvider theme={createTheme()}>
        <ButtonToolbar />
      </MuiThemeProvider>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
