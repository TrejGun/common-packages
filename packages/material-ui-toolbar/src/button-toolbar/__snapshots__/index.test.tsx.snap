// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonToolbar /> renders component 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-justify-content-xs-flex-end"
  />
</DocumentFragment>
`;
